tests/std/tests/GH_004618_mixed_operator_usage_keeps_statistical_properties/test.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cmath>
+#include <cstddef>
+#include <numeric>
+#include <random>
+#include <vector>
+
+
+static const size_t N = 1000000;
+
+void check_results(std::vector<double> const& generated, double expected_mean, double expected_variance) {
+    const double actual_mean = std::accumulate(generated.begin(), generated.end(), 0.0) / generated.size();
+    assert(std::abs((actual_mean - expected_mean) / expected_mean) < 0.01);
+
+    double actual_variance = 0;
+    double actual_skew     = 0;
+    for (auto value : generated) {
+        auto deviation = value - actual_mean;

```

Comment:
Nitpicks: We should iterate in-place (`const auto&`) and we can mark `deviation` as `const` too.